src/test/scala/uk/gov/homeoffice/cirium/BaseSpecification.scala

```diff
@@ -0,0 +1,9 @@
+package uk.gov.homeoffice.cirium
+import github.gphat.censorinus.StatsDClient
+import org.specs2.mutable.Specification
+
+trait BaseSpecification extends Specification {
+  val statsDClient: StatsDClient = new StatsDClient(hostname = AppConfig.statsdHost, port = AppConfig.statsdPort, prefix = AppConfig.statsdPrefix)
```

Comment:
It would be good to wrap the statsdclient in our own class that can be mocked so we don't create statsd connections and send stats during tests